torch/_functorch/aot_autograd.py

```diff
@@ -2257,6 +2299,51 @@ def runtime_wrapper(*args):
             return fw_outs
     return runtime_wrapper
 
+def get_live_fake_mode_from_dispatch():
+    modes = torch.utils._python_dispatch._get_current_dispatch_mode_stack()
```

Comment:
I think Ed recently added a util to do this more soundly: check out `detect_fake_mode()` in `torch/_guards.py`

Reply:
Thanks!